ts/features/itwallet/identification/screens/spid/ItwSpidIdpLoginScreen.tsx

```diff
@@ -97,40 +98,42 @@ const ItwSpidIdpLoginScreen = () => {
 
   const content = useMemo(
     () =>
-      O.fold(
-        () => LoadingSpinner,
-        (url: string) => (
-          <WebView
-            cacheEnabled={false}
-            androidCameraAccessDisabled
-            androidMicrophoneAccessDisabled
-            javaScriptEnabled
-            textZoom={100}
-            originWhitelist={originSchemasWhiteList}
-            source={{ uri: url }}
-            onError={onError}
-            onHttpError={onError}
-            onNavigationStateChange={handleNavigationStateChange}
-            onShouldStartLoadWithRequest={handleShouldStartLoading}
-            allowsInlineMediaPlayback
-            mediaPlaybackRequiresUserAction
-            userAgent={defaultUserAgent}
-          />
-        )
-      )(spidAuthUrl),
+      !isMachineLoading && O.isSome(spidAuthUrl) ? (
+        <WebView
+          cacheEnabled={false}
+          androidCameraAccessDisabled
+          androidMicrophoneAccessDisabled
+          javaScriptEnabled
+          textZoom={100}
+          originWhitelist={originSchemasWhiteList}
+          source={{ uri: spidAuthUrl.value }}
+          onError={onError}
+          onHttpError={onError}
+          onNavigationStateChange={handleNavigationStateChange}
+          onShouldStartLoadWithRequest={handleShouldStartLoading}
+          allowsInlineMediaPlayback
+          mediaPlaybackRequiresUserAction
+          userAgent={defaultUserAgent}
+          onLoadEnd={onLoadEnd}
+        />
+      ) : (
+        <></>
+      ),
```

Comment:
If `spidAuthUrl` is not ready or present, it likely indicates that the machine is still loading. This makes `isMachineLoading` redundant in this context. I suggest we keep the original code and add a fallback to `null` to prevent rendering multiple spinners. What do you think?

```suggestion
    O.fold(
        () => null,
        (url: string) => (
          <WebView
            key={"spid_webview"}
            cacheEnabled={false}
            androidCameraAccessDisabled
            androidMicrophoneAccessDisabled
            javaScriptEnabled
            textZoom={100}
            originWhitelist={originSchemasWhiteList}
            source={{ uri: url }}
            onError={onError}
            onHttpError={onError}
            onNavigationStateChange={handleNavigationStateChange}
            onShouldStartLoadWithRequest={handleShouldStartLoading}
            allowsInlineMediaPlayback
            mediaPlaybackRequiresUserAction
            userAgent={defaultUserAgent}
            onLoadEnd={onLoadEnd}
          />
        )
      )(spidAuthUrl),
```

Reply:
I agree that having `isMachineLoading` set to true and having the auth URL at the same time is unlikely, as one depends on the other. The machine is loading first, and then it receives the auth URL. Therefore, `isMachineLoading` is redundant in this context.
I'm agree with you